image.yml

```diff
@@ -24,6 +26,12 @@
         state: started
         enabled: true
 
+   - name: Enable osbuild-composer socket
```

Comment:
The task right before this already does it, just used `ansible.builtin.systemd` instead of `ansible.builtin.service`. Do we want to change to `ansible.builtin.service`, which is more generic? Keep in mind that we require a RHEL system anyway.

Reply:
Ah sorry, I think you added that after I forked. I will just remove mine.